scripts/setup_linux.sh

```diff
@@ -102,6 +102,14 @@ print_warning() {
   printf "\e[0;33m%s\e[0m" "${1}"
 }
 
+# Verify this script is running from the correct folder (root directory)
+dir_err_txt="Please run this script only from the root of the garage \
+repository, i.e. you should run it using the command \
+\"bash scripts/setup_linux.sh\""
+([ -e setup.py ] && (grep -Fq "name='rlgarage'," setup.py)) \
```

Comment:
I don't think this conforms to the style guide for conditionals
https://google.github.io/styleguide/shell.xml#Test,_%5B_and_%5B%5B

@gonzaiva opinion?

Reply:
@ryanjulian I wanted to keep code consistency with what existed already. I can change all the conditionals to if statements if that's better.

Reply:
I don't see anywhere else in this file which uses the subshell `()` style you used here for a conditional statement. everything below either uses `test -f` or `if [[  ]]``

Reply:
Yes, please use test or if. Makes things more verbose but also clearer. Also, I think a better way to verify this is using the current working directory (pwd): if we're executing this script one level down the path we're good. The problem in relying in setup.py is that if in the future we move/rename the file (not likely but it could happen), nobody is going to remember to update this verification as well.
Please also remove/update my failed code for this :(
https://github.com/rlworkgroup/garage/blob/dfba4206c8d938d41a592e9fd8ce1f16f4aab1c6/scripts/setup_linux.sh#L109

Reply:
Okay, but I think for our case it would be sufficient to check if we're running the script one directory below. However, it's true that if we move the installation script, my suggestion will fail.

Reply:
What if I execute the script from `/`? What if I execute it from `/home/rjulian/code/notgarage` but there is a directly `notgarage/scripts`?

Reply:
If we use `$(dirname ${0})` in the script, we get the relative path from our current working directory to the script, which should be `./scripts`, and we don't rely on the garage folder or the root, but I accept the setup.py solution as well.

Reply:
Angel's solution is fine with me, too.

Reply:
Code has been changed to use test in an if statement for clarity. 👍